src/Css/Style.php

```diff
@@ -1640,6 +1688,12 @@ protected function compute_prop(string $prop, $val)
             return $val;
         }
 
+        // Resolve CSS custom property value.
+        if (substr($val, 0, 4) === 'var(') {
+            $var = substr(trim($val), 4, -1);
+            return $this->parse_var($var);
```

Comment:
We should allow normal computation after this so we can accommodate something like:

```
<style>
    :root {
        --half: 50%;
    }
    div {
        width: var(--half);
    }
</style>
<div>dompdf</div>
```

Reply:
The property is parsed before the value is computed, so I think this should work... Will add a test for it though.

Reply:
I think we're good now.